cmd/utils/flags.go

```diff
@@ -504,10 +506,22 @@ var (
 		Usage: "Allow insecure account unlocking when account-related RPCs are exposed by http" + generateEnvDoc(c_NodeFlagPrefix+"allow-insecure-unlock"),
 	}
 
-	CoinbaseAddressFlag = Flag{
-		Name:  c_NodeFlagPrefix + "coinbases",
+	QiCoinbaseFlag = Flag{
```

Comment:
we should do this using one flag, why do we technically need these two flags ?

Reply:
I'm good with combining them, this way makes it more explicit but potentially less fool-proof